syscall/linux/types.lua

```diff
@@ -1016,6 +1016,74 @@ mt.cpu_set = {
 
 addtype(types, "cpu_set", "struct cpu_set_t", mt.cpu_set)
 
+local ulong_bit_count = ffi.sizeof('unsigned long') * 8
+local function ulong_index_and_bit(n)
+  local i = math.floor(n / ulong_bit_count)
+  local b = bit.lshift(1ULL, n - i * ulong_bit_count)
```

Comment:
I think that may be luajit 2.1 vs 2.0 issue, about 64 bit bitops. Would be nice to work around...